setup.py

```diff
@@ -162,6 +167,10 @@ def run(self):
         "comtypes.tools",
         "comtypes.test",
     ],
+    # Preserve Python 3 compatibility during building & installation with pip.
+    setup_requires=[SETUPTOOLS_PY3_SUPPORT,],
+    # Also trick pip to put the forked package replacement on top of the official one.
+    install_requires=[SETUPTOOLS_PY3_SUPPORT, "comtypes"],
```

Comment:
Usually only this is taken into account by `pip install .`.

Reply:
The `setup_requires` should affect when building the wheel, if I'm not completely incorrect.

Reply:
Should, but for some reason it wasn't enough. The winning recipe is to specify the **setuptools** pin also in `install_requires` and install it with `pip install` as `setup.py install` fails to do the right thing. (because pip gathers all deps prior to run the setup install phase)

Anyway, this is the safest way to ensure we get the right syntax :D.